blog for nav and dashboard (#1722)
pulumi/pulumi-hugo: themes/default/content/blog/new-improved-pulumi-service-user-experience/index.md

# The meta_image appears in social-media previews and on the blog home page.
# A placeholder image representing the recommended format, dimensions and aspect
# ratio has been provided for you.
meta_image: meta.png

# At least one author is required. The values in this list correspond with the `id`
# properties of the team member files at /data/team/team. Create a file for yourself
# if you don't already have one.
authors:
    - susan-evans

# At least one tag is required. Lowercase, hyphen-delimited is recommended.
tags:
    - pulumi-service
    - user-experience

# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
# for additional details, and please remove these comments before submitting for review.
---

It’s been nearly four years since the Pulumi Service console launched back in 2018. The [Pulumi Service]({{< relref "/product/pulumi-service" >}}) is the easiest way to use Pulumi’s open source, universal infrastructure as code SDK at scale and provides a fully managed experience. It handles infrastructure state and secrets, sets up SAML SSO, integrates with CI/CD pipelines, and enforces compliance rules.

> In these last four years, the Service’s features and capabilities have grown quickly. We’ve added features like stack tags, stack changes view, and resource visualizations. But throughout all these changes, the look and feel of the Service has largely stayed the same—until now!

Author: @lukehoban @meagancojocar can yall suggest what you want in this feature list in the second sentence? thank you!

Author: cool, guess i guess my random selection was a good one, heh

> In these last four years, the Service’s features and capabilities have grown quickly. We’ve added features like stack tags, stack changes view, and resource visualizations. But throughout all these changes, the look and feel of the Service has largely stayed the same—until now!

Reviewer: ```suggestion
In these last four years, our customer base has grown significantly Service’s features and capabilities have grown quickly. We’ve added features like audit logs, SAML SSO and SCIM, stack tags, resource visualizations, and much more! But throughout all these changes, the look and feel of the Service has largely stayed the same—until now!
```

<!--more-->

Today we’re excited to launch a set of improvements to the overall user experience of the Pulumi Service. These updates include:

> - Easier navigation around the Service

Author: This blog was originally written with a usability focus that has since changed to a product focus, so I have zero strong opinions about any of it. Feel free to suggest what you want here and I'll accept it.

Among the biggest changes to the feel of the Service is our new left navigation pane. Navigation items like your Dashboard, Stacks, and Settings are accessible in the new pane from anywhere in the Service.

![""](dashboard-nav-expanded.png)

The navigation pane consolidates actions and information so it's easy for you to find common items. You'll find options in the pane to:

- switch between your individual account and any other organizations you're part of
- create new organizations
- access your favorite stacks (read on to find out more!)
- directly access sub-pages like Billing or Access Tokens
- send us your feedback!

These changes will help you move through the Pulumi Service without losing your place and to ensure convenient access to key pages. You can even collapse the navigation pane for more screen real estate and still access any page you need.

![""](dashboard-nav-collapsed.png)

If you're looking for your account details, they have moved into the black utility toolbar. To access your account settings or logout of the Pulumi Service, select your profile image at the top right of your browser window.

## Stack Favorites

> Many Pulumi users have access to dozens of hundreds of stacks. But only a few of those stacks may require regular interaction. We've introduced a new Stack Favorites feature to let you flag the stacks that are most important. This way you can access those stacks from anywhere in the Service.

Reviewer 2: ```suggestion
Many Pulumi users have access to dozens or hundreds of stacks. But only a few of those stacks may require regular interaction. We've introduced a new Stack Favorites feature to let you flag the stacks that are most important. This lets you access those stacks from anywhere in the Service.
```

> Your favorites list is your own. Even if you're part of an organization, your favorites won't be shared with anyone other than you. And you can have as many favorites as you like!

Reviewer 2: ```suggestion
Your favorites list is your own. Even if you're part of an organization, your favorites won't be shared with anyone else. And you can have as many favorites as you like!
```
